admin-ui/src/pages/models-page.tsx

```
@@ -0,0 +1,514 @@
+import { useCallback, useEffect, useMemo, useState } from "react"
+import { useTranslation } from "react-i18next"
+import { toast } from "sonner"
+
+import {
+  AdminApiError,
+  type AdminModelDetailsItem,
+  getAdminModelDetails,
+} from "@/lib/admin-api"
+import { i18n } from "@/lib/i18n"
+import { cn } from "@/lib/utils"
+import { Badge } from "@/components/ui/badge"
```

ESLint

[error] Expected "@/components/ui/badge" to come before "@/lib/utils".

(perfectionist/sort-imports)